[SOAR-1882] Insight IDR plugin - add set user action (#622)
rapid7/insightconnect-plugins: rapid7_insightidr/help.md

The connection configuration accepts the following parameters:

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|api_key|credential_secret_key|None|True|InsightIDR API key|None|None|
|url|string|https://us.api.insight.rapid7.com|True|The URL endpoint for InsightIDR. e.g. https://<REGION_CODE>.api.insight.rapid7.com|None|None|

Example input:

> ```

Reviewer: Let's add the connection example with a fake key that matches the format

```

## Technical Details

### Actions

#### Set Status of Investigation

> This action is used to set the status of the investigation with the given ID.

Reviewer: ```suggestion
This action is used to set the status of the investigation by the Investigation ID.
```

##### Input

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|id|string|None|True|The ID of the investigation to change the status of|None|None|
|status|string|CLOSED|True|The new status for the investigation|['OPEN', 'CLOSED']|None|

Example input:

> ```

Reviewer: Need example JSON input

```

##### Output

|Name|Type|Required|Description|
|----|----|--------|-----------|
|investigation|investigation|True|The investigation for which the status was set|

Example output:

> ```

Reviewer: Add example JSON output
